edsnlp/pipelines/misc/sections/patterns.py

```diff
@@ -95,6 +99,17 @@
 
 traitements_sortie = [r"traitement de sortie"]
 
+evolution = [r"evolution", r"evolution et examen clinique aux lits portes :"]
+
+modalites_sortie = [r"modalites de sortie", r"devenir du patient"]
+
+vaccinations = [r"vaccinations", r"vaccination"]
+
+introduction = [
+    r"compte rendu d'hospitalisation du \d{2}/\d{2}/\d{4} au \d{2}/\d{2}/\d{4}",
+    r"compte rendu d'hospitalisation .{0,25}",
+]
+
```

Comment:
I think my initial proposal could be improved by suppressing the space in " .{0,25}", changing "compte rendu" to "compte.?rendu", and also be made less redundant in the following way :
```suggestion
introduction_CR = [
    r"compte.?rendu d'hospitalisation.{0,30}",
]

```
Do you think it's ok ?

Reply:
Great !